DataViews/Program.cs

```diff
@@ -229,18 +234,17 @@ public static async Task<bool> MainAsync(bool test = false)
                 #region step7
                 Console.WriteLine("Step 7: View Fields Available to Include in the Data View");
 
-                ResolvedItems<FieldSet> availableFields = await dataviewService.GetAvailableFieldSetsAsync(dataView.Id).ConfigureAwait(false);
+                var availableFields = await dataviewService.GetAvailableFieldSetsAsync(dataView.Id).ConfigureAwait(false);
 
                 Console.WriteLine();
                 Console.WriteLine($"Available fields for data view {dataView.Name}:");
                 foreach (var fieldset in availableFields.Items)
                 {
-                    Console.WriteLine($"Fieldset:");
-                    Console.WriteLine($"QueryId: {fieldset.QueryId}");
-                    Console.WriteLine($"Data Fields: ");
+                    Console.WriteLine($"  QueryId: {fieldset.QueryId}");
```

Comment:
is the idea for the space to show this is a part of the dataview from ln 240?

Reply:
the idea was to nest the output to imply arrays, yea. the data view can have multiple fieldsets (this one only has one), so it's offset 1 tab for the query and data fields list header, then the individual data fields are also an array so they're offset 1 more tab from their header. I tried it originally with no tabbing and thought it looked really bad. I can always put it back, especially since there's only one fieldset, but I like this formatting a little better

Reply:
makes sense to me.  i didn't run it, but judging from the code it sounds good